actors/power/src/lib.rs

```diff
@@ -364,6 +367,32 @@ impl Actor {
         })
     }
 
+    /// Returns the total raw power of the network.
```

Comment:
As a public API method, this needs more documentation.

Raw byte power is the sum of the active (i.e. non-faulty) byte commitments of all miners which have more than the consensus minimum amount of storage active.

Reply:
You're right, thank you.